packages/api/test/unit/beacon/oapiSpec.test.ts

```diff
@@ -84,11 +84,90 @@ const testDatas = {
   ...validatorTestData,
 };
 
+const ignoredOperations = [
```

Comment:
Just looking at this, it is not immediately clear how it works. Maybe it makes sense to have separate ignore lists
- ignoredRoutes
- ignoredReqParams
- ignoredResParams

or similar

Reply:
I will clarify that. As those are per operation (and not global), it felt more natural to have some operation level extra data.